test/maps/lua_testsuite.wmf/scripting/road.lua

```diff
@@ -18,7 +18,7 @@ function road_tests:setup()
 
    self.start_flag = player1:place_flag(self.f)
 
-   self.r = player1:place_road(self.start_flag, "r", "r", "br", "br")
+   self.r = player1:place_road("normal", self.start_flag, "r", "r", "br", "br")
```

Comment:
How about placing a busy road here to give it test coverage?

Code LGTM otherwise, not tested yet.